insights/parsers/sysconfig.py

```diff
@@ -24,36 +36,76 @@
 LibvirtGuestsSysconfig - file ``/etc/sysconfig/libvirt-guests``
 ---------------------------------------------------------------
 
+MemcachedSysconfig - file ``/etc/sysconfig/memcached``
+------------------------------------------------------
+
 MongodSysconfig - file ``/etc/sysconfig/mongod``
 ------------------------------------------------
 
+NetconsoleSysconifg -file ``/etc/sysconfig/netconsole``
```

Comment:
Should be `NetconsoleSysconfig`.

Reply:
@bfahr - Thank you for the careful review. They are fixed.